network-api/networkapi/wagtailpages/migrations/0023_add_localizable_field_names.py

```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('wagtailpages', '0022_product_voting'),
+        ('wagtailpages', '0019_pni_snippet_conversion'),
```

Comment:
Dependency change from merged migrations

Reply:
this got updated in https://github.com/mozilla/foundation.mozilla.org/pull/5623/files already so we don't need that here.